src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -176,65 +177,82 @@
                                           :action-fn           #(re-frame/dispatch [:accounts.logout.ui/logout-pressed])}]]]]))
 
 (defview advanced-settings [{:keys [network networks dev-mode? settings]} on-show]
-  {:component-did-mount on-show}
-  [react/view
-   (when (and config/extensions-enabled? dev-mode?)
+  (letsubs [supported-biometric-auth [:get-supported-biometric-auth]]
```

Comment:
looks like there is some changes here that are not related to the PR? Maybe a merge accident?

Reply:
@yenda are you referring to deleted lines here?

Reply:
btw that's very strange.. let me check and fix

Reply:
@yenda deleted lines are actually there (after the sub), it is a diff glitch

![image](https://user-images.githubusercontent.com/15999009/58097879-aac99200-7bd8-11e9-95f7-76c76634c269.png)

Reply:
sorry for the late reply, even though this seems compliant with how the defview macro works I think it would be better to turn this view into a simple defn and subscribe to the subscription in the parent, passing only the value as parameter of this component, there is no compelling reason for this view to use the defview macro

Reply:
@yenda I did a test and seems to me that if I make it defn when pushing "Advanced ^" the view doesn't scroll down anymore... I leave it a defview but I move subscription in parent.. make sense?